bitwarden_license/src/Scim/Commands/Users/GetUserCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Scim.Commands.Users.Interfaces;
+using Bit.Scim.Models;
+
+namespace Bit.Scim.Commands.Users;
+
+public class GetUserCommand : IGetUserCommand
```

Comment:
In the CQRS pattern, commands are write operations and queries are read operations.

This and all other "get" operations should be called queries (e.g. `GetUserQuery`) in the `Bit.Scim.Queries` namespace.

Reply:
I created `GetUserQuery` under the namespace `Bit.Scim.Queries.Users`.